handbook.md

```diff
@@ -236,6 +236,61 @@ As a maintainer, your responsibility is to keep a project running, taking care o
 
 Traditionally, the process to become a maintainer has been simple. Take care of something for long enough and people will expect you to maintain it. We might ask you directly if you want to be a maintainer if you're maintaining something for some time. If you have made contributions only on a very specific topic (an Operating System, for instance), we might ask you to be a maintainer for that part of the code.
 
+## Becoming a committer
+#### What is a committer
+
+A Foreman committer is a person with commit access to one or more of the repositories under [the Foreman organization](https://github.com/theforeman/). Committers are members of the Foreman contributors community who exhibit most of the following behaviors:
+
+* Review and merge code and documentation
+* Help triaging bugs and testing pull requests
+* Make well formed pull requests
+* Take care of releases
+* Have a sense of duty about the Foreman project
+* Play well with others, are respectful, show gratitude
+
+We ask new Foreman contributors to act as a committer to the extent capable. It's the best route to become a committer if that is your goal, but even if it's not, it makes our community better.
+
+If you become a committer, we expect of you to:
+
+*  Help out users and other developers on the mailing lists, in Redmine, Serverfault, and in IRC #theforeman and #theforeman-dev
```

Comment:
It now sounds like you need to do all those things, but IMHO you don't have to do all. For example, I'm not on Serverfault and not on the users mailing list.